### PR TITLE
drainer: Refactor translator/kafka.go to make it simpler

### PR DESCRIPTION
**Pull request by suzaku**
### What problem does this PR solve? 

Make `TiBinlogToSlaveBinlog` simpler.

### What is changed and how it works?

Refactor by extracting functions from `TiBinlogToSlaveBinlog`.

### Check List 

Tests 

 - Unit test

Code changes

Side effects

Related changes

### CODE REVIEW

File: drainer/translator/kafka.go [hunk 1]
```diff
@@ -45,81 +50,51 @@ func TiBinlogToSlaveBinlog(infoGetter TableInfoGetter, schema string, table stri
...
-					tableMutation.Type = obinlog.MutationType_Delete.Enum()
-					tableMutation.Row, err = deleteRowToRow(info, row)
-					if err != nil {
-						return nil, err
-					}
-				default:
-					return nil, errors.Errorf("unknown mutation type: %v", mutType)
+		iter := newSequenceIterator(&mut)
+		for {
+			table, err := nextTable(schema, info, iter)
```
Comment (WangXiangUSTC) [inline drainer/translator/kafka.go]: `nextTable` seems confused, how about put `mutType, row, err := iter.next()` out of this function, and change to a new function name?

Comment (GregoryIan) [reply drainer/translator/kafka.go]: `nextRow`

**Comment by GregoryIan** [comment]
Rest LGTM

File: drainer/translator/kafka.go (hunk 1 above)
Comment (suzaku) [reply drainer/translator/kafka.go]: @WangXiangUSTC @GregoryIan Updated.

**Comment by GregoryIan** [comment]
LGTM

**Pull request merged**